packages/sample/src/main.ts

```diff
@@ -1,11 +1,13 @@
 import {Command} from "commander";
 import {ExpansionHub, getConnectedExpansionHubs} from "@rev-robotics/expansion-hub";
 import {RevHub} from "@rev-robotics/expansion-hub";
+import { DIODirection } from "@rev-robotics/rhsplib";
 
 const program = new Command();
 
 program.version('1.0.0')
     .option('-l --list', 'List connected devices')
+    .option('-b --button', 'Run button press demo')
```

Comment:
We should be exposing primitives via the CLI, not demos.

Reply:
Changed it to use commands